File: auto-docs/executables/r/line-scatter.r
library(plotly)

p <- plotly(username='theengineear', key='o9zlr0hy6z')

trace1 <- list(
  x = c(1, 2, 3, 4), 
  y = c(10, 15, 13, 17), 
  mode = "markers", 
  type = "scatter"
)
trace2 <- list(
  x = c(2, 3, 4, 5), 
  y = c(16, 5, 11, 9), 
  mode = "lines", 
  type = "scatter"
)
trace3 <- list(
  x = c(1, 2, 3, 4), 
  y = c(12, 9, 15, 12), 
  mode = "lines+markers", 
  type = "scatter"
)
data <- list(trace1, trace2, trace3)

response <- p$plotly(data, kwargs=list(filename="line-scatter", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/node_js/line_scatter.js
// Learn about API authentication here: {BASE_URL}/nodejs/getting-started
// Find your api_key here: {BASE_URL}/settings/api

var plotly = require('plotly')('TestBot', 'r1neazxo9w');
var trace1 = {
  x: [1, 2, 3, 4], 
  y: [10, 15, 13, 17], 
  mode: "markers", 
  type: "scatter"
};
var trace2 = {
  x: [2, 3, 4, 5], 
  y: [16, 5, 11, 9], 
  mode: "lines", 
  type: "scatter"
};
var trace3 = {
  x: [1, 2, 3, 4], 
  y: [12, 9, 15, 12], 
  mode: "lines+markers", 
  type: "scatter"
};
var data = [trace1, trace2, trace3];
var graphOptions = {filename: "line-scatter", fileopt: "overwrite"};
plotly.plot(data, graphOptions, function (err, msg) {
    console.log(msg);
});
